.github/workflows/stale.yaml

```
@@ -0,0 +1,68 @@
+"on":
+  schedule:
+  - cron: "30 1 * * *"
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: 'actions/stale@v7'
```

actionlint (1.7.4)

the runner of "actions/stale@v7" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)